### ISSUE TITLE
[1.6.3] client shows incorrect cipher info "encrypted with 0 bit"

### ISSUE THREAD

**Issue by Wikinaut** [owncloud/client#1436]
- 1.5.1
- 1.5.2 [UPDATED 28.02.2014]
- 1.5.3rc1 [UPDATED 05.03.2014]
- 1.6.0 [UPDATED 15.06.2014]
- 1.6.1rc1 [UPDATED 19.07.2014]
  clients on 
- Windows 8 and 
- Linux OpenSuse 13.1

show "0 bit" when hovering on the padlock (**the certificate info is shown correctly, when clicking onto the padlock**)

<img src="http://i.imgur.com/KwUfHMs.png">

```
openssl s_client -connect www.example.com:443
```

gives

```
CONNECTED(00000003)
depth=0 CN = *.example.com
verify error:num=20:unable to get local issuer certificate
verify return:1
depth=0 CN = *.example.com
verify error:num=27:certificate not trusted
verify return:1
depth=0 CN = *.example.com
verify error:num=21:unable to verify the first certificate
verify return:1

---
Certificate chain
 0 s:/CN=*.example.com
   i:/O=CAcert Inc./OU=http://www.CAcert.org/CN=CAcert Class 3 Root

---
Server certificate
-----BEGIN CERTIFICATE-----
MIIF5zCCA8+gAwIBAgIDAiFQMA0GCSqGSIb3DQEBDQUAMFQxFDASBgNVBAoTC0NB
...
nDR3+6adGb2V9LEPBSe0OSMMnFuGxqO21Kd/9h3xD04qUj6jHYIANqjJzbjHCQgR
Cw/qn+IEb+umi3y5iT6pglQlxm6vn2p7FAEw
-----END CERTIFICATE-----
subject=/CN=*.example.com
issuer=/O=CAcert Inc./OU=http://www.CAcert.org/CN=CAcert Class 3 Root

---
No client certificate CA names sent

---
SSL handshake has read 1851 bytes and written 889 bytes

---
New, TLSv1/SSLv3, Cipher is RC4-SHA
Server public key is 4096 bit
Secure Renegotiation IS supported
Compression: NONE
Expansion: NONE
SSL-Session:
    Protocol  : TLSv1
    Cipher    : RC4-SHA
    Session-ID: 925CCF0FAEB16B4FF06A1C................
    Session-ID-ctx: 
    Master-Key: 5CE1711DE7510B8B2DAC58................
    Key-Arg   : None
    PSK identity: None
    PSK identity hint: None
    SRP username: None
    TLS session ticket:
    0000 - 13 a8 1d 92 45 a5 62 47-9f 8a 4e ac 8d 24 0e 8d   ....E.bG..N..$..
    ......
    00b0 - 9e 82 5f 04 90 03 10 94-18 30 ff 12 43 75 0e 3a   .._......0..Cu.:

    Start Time: 1392670954
    Timeout   : 300 (sec)
    Verify return code: 21 (unable to verify the first certificate)

```

**Comment by Wikinaut** [comment]
This issue is now finally solved in version >> 2.0.0 << !!

**Comment by guruz** [comment]
:-) Good to hear. Qt was somehow buggy so we worked around it by fetching the returned SSL info at several times and merging it to get a good state in the UI
